pom.xml

```diff
@@ -144,7 +144,7 @@
             <dependency>
                 <groupId>com.okta.oidc.tck</groupId>
                 <artifactId>okta-oidc-tck</artifactId>
-                <version>0.5.8</version>
+                <version>0.5.9-SNAPSHOT</version>
```

Comment:
TODO for @arvindkrishnakumar-okta: Update to `0.5.9` after https://github.com/okta/okta-oidc-tck/pull/169 is merged and `0.5.9` is released.